src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      data-qa="big"
+      method="post"
+      class="wrapper"
+    >
+      <label
+        for="search-input-big"
+        class="search-input"
+      >
+        <input
+          class="form-input"
+          type="text"
+          id="search-input-big"
+          name="search-input-big"
```

Comment:
You should use camelCase for your variables
![image](https://user-images.githubusercontent.com/96409563/195801943-c4c302b9-4dd2-47cd-9dd3-443a2892c390.png)
Also missing `data-qa="keypress"` attribute
![image](https://user-images.githubusercontent.com/96409563/195802825-db7ff177-33b9-4988-9033-b84a41da5f30.png)